pickhardtpayments/Payment.py

```diff
@@ -0,0 +1,207 @@
+import logging
+import time
+
+import Attempt
+
+
+class Payment:
+    """
+    Payment stores the information about an amount of sats to be delivered from source to destination.
+
+    When sending an amount of sats from sender to receiver, a payment is usually split up and sent across
+    several paths, to increase the probability of being successfully delivered.
+    The PaymentClass holds all necessary information about a payment.
+    It also holds information that helps to calculate performance statistics about the payment.
+
+    :param sender: sender address for the payment
+    :type sender: class:`str`
+    :param receiver: receiver address for the payment
+    :type receiver: class:`str`
+    :param total_amount: The total amount of sats to be delivered from source address to destination address.
+    :type total_amount: class:`int`
+    """
+
+    def __init__(self, sender, receiver, total_amount: int = 1):
+        """Constructor method
+        """
+        self._successful = False
+        self._ppm = -1
```

Comment:
Not sure if the pythonic way would be to use `None`?

Reply:
Regarding `self._succesful`? Hm, I was unsure myself. 
I then went for `False`, and my thinking was: `None` would imply, that I don't have any information at instantiation. However, I do now, that the Payment was not yet successful. Consequently, I could send it to reflect this as a state.

I did change the other three attribute values. :)